pkg/schema/schema.go

```diff
@@ -62,6 +62,7 @@ type TemplatesSettingsGomplate struct {
 type Terraform struct {
 	BasePath                string `yaml:"base_path" json:"base_path" mapstructure:"base_path"`
 	ApplyAutoApprove        bool   `yaml:"apply_auto_approve" json:"apply_auto_approve" mapstructure:"apply_auto_approve"`
+	AppendUserAgent         string `yaml:"append_user_agent" json:"append_user_agent" mapstructure:"append_user_agent"`
```

Comment:
How will this help us set this?


 TF_APPEND_USER_AGENT="Cloud Posse Atmos v1.181 (https://atmos.tools)"

Reply:
We should be able to override it, but it should default to this UA.

Reply:
I will push a set to default for this env `Cloud Posse Atmos v1.181 (https://atmos.tools/)`

Reply:
We should also consider respecting the env var set out side of atmos

Reply:
this is done